src/go/rpk/pkg/cli/cmd/wasm/template/package_json.go

```diff
@@ -11,7 +11,7 @@ package template
 
 const packageJson = `{
   "name": "wasm-panda",
-  "version": ">= 21.8.2",
+  "version": "21.8.2",
```

Comment:
@graphcareful What version do I need to use here?

Reply:
This is a template, so it should fill in the version that rpk wasm generate discovers when it runs `npm search vectorized-wasm`

Reply:
This is the version of `wasm-panda` itself, which is a demo program. This can be anything I would pick `0.0.1` but the one thing that cannot be here are any symbols like `>=` because they don't work and don't make sense in this context. Those symbols should only be used in the `dependencies` portion